R/plots.r

```diff
@@ -695,9 +695,13 @@ setMethod("plotRap", "list", function(object,facet=TRUE,
 
 makeRegiPlot <- function(regiScoresFrame){
   regiScoresFrame[,"GenomicIntervals"] <- factor(regiScoresFrame[,"GenomicIntervals"],levels=unique(as.vector(regiScoresFrame[,"GenomicIntervals"])))
-  Plot <- ggplot(regiScoresFrame, aes(Sample,GenomicIntervals))  
+  Plot <- ggplot(regiScoresFrame, aes(GenomicIntervals,Sample))  
   Plot <- Plot+geom_tile(aes(y=Sample,x=GenomicIntervals,fill = log2_Enrichment)) +
-    scale_fill_gradient2(low="blue",high="yellow",mid="black",midpoint=median(regiScoresFrame$log2_Enrichment))
+    scale_fill_gradient2(low="blue",high="yellow",mid="black",midpoint=median(regiScoresFrame$log2_Enrichment)) +
+    theme(axis.text.x = element_text(angle = 90)) +
```

Comment:
Would adding `hjust = 1` (right) align text to the ticks?

Alternatively, you could dodge the labels (see [here](https://stackoverflow.com/a/60650595/322912)).

Reply:
`hjust = 1` did not solve this but using `scale_x_discrete(guide = guide_axis(angle = 90))` did the trick. Thanks!